src/database/DataTypes/Item.js

```diff
@@ -225,7 +225,7 @@ export class Item extends Realm.Object {
    */
   getBatchesInLocation({ id: locationId } = {}) {
     if (!locationId) return this.batches;
-    return this.batches.filtered('location.id = $0', locationId);
+    return this.batches.filtered('location.id = $0 & numberOfPacks > 0', locationId);
```

Comment:
do wee need to filter by numberOfPacks first, and then return either this.batches or batches filtered by location ?

Reply:
Good point. Used an existing helper method also instead of explicit query: 8ea020f5999f806219c2e1df4b733f7a6a5aa8a9 not sure if that's the best way to go, but I imagine it's equivalent in the long run

Reply:
great